docs/earthfile/earthfile.md

```diff
@@ -648,10 +648,28 @@ For detailed examples demonstrating how other scenarios may function, please see
 
 #### Description
 
-In the *output form*, the command `SAVE IMAGE` marks the current build environment as the image of the target and assigns an output image name.
+In the *output form*, the command `SAVE IMAGE` marks the current build environment as the image of the target and assigns one or more output image names.
 
 In the *cache hint form*, it instructs Earthly that the current target should be included as part of the explicit cache. For more information see the [shared caching guide](../guides/shared-cache.md).
 
+{% hint style='info' %}
+##### Assigning more than one image name
```

Comment:
In addition to being more concise, this also lets a user find this while searching for "multiple". 
```suggestion
##### Assigning multiple image names
```